ulib/axlibc/src/sys.rs

```diff
@@ -7,13 +7,40 @@
  *   See the Mulan PSL v2 for more details.
  */
 
-use arceos_posix_api::sys_sysconf;
+use arceos_posix_api::{config, ctypes, sys_getrlimit};
```

Comment:
Use `crate::ctypes`. Please check how we wrap two different `ctypes` in `axlibc/src/lib.rs`